src/Compilers/CSharp/Portable/Symbols/Attributes/AttributeData.cs

```diff
@@ -834,6 +834,12 @@ internal static CSharpSyntaxNode GetAttributeArgumentSyntax(this AttributeData a
             return arguments.Length == 1 && arguments[0].TryDecodeValue(SpecialType.System_String, out string? value) ? value : null;
         }
 
+        internal static string? DecodeCallerArgumentExpressionAttribute(this CSharpAttributeData attribute)
```

Comment:
>DecodeCallerArgumentExpressionAttribute [](start = 32, length = 39)

It looks like there is only one call site, I think it is not worth adding this helper to this type.

Reply:
@AlekseyTs Will probably need it for VB if it was decided to support it.

Reply:
>Will probably need it for VB if it was decided to support it.

This is a C# specific type.

---
In reply to: [600294828](https://github.com/dotnet/roslyn/pull/51952#discussion_r600294828) [](ancestors = 600294828)